src/main/java/seedu/unify/logic/commands/DeleteCommand.java

```diff
@@ -52,7 +53,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         // Deletes from the last item to prevent future deletes operating on wrong indexes
         for (int i = targetIndexes.size() - 1; i >= 0; i--) {
-            if (targetIndexes.get(i).getZeroBased() >= lastShownList.size()) {
+            if (targetIndexes.get(i).getZeroBased() >= lastShownList.size() || targetIndexes.get(i).getOneBased() < 1) {
```

Comment:
Shouldn't just using either getZeroBased() or getOneBased() make the condition more consistent to make it easier to check the condition as a developer.

Reply:
You're right, that was actually a previous implementation. Forgot to remove it before pushing the code.
Will change it.